forloop_modules/function_handlers/control_flow_handlers.py

```diff
@@ -144,31 +149,46 @@ def export_imports(self, *args):
         return (imports)
 
 
-class RunPipelineHandler(AbstractFunctionHandler):
+class SchedulePipelineHandler(AbstractFunctionHandler):
     def __init__(self):
-        self.icon_type = "RunPipeline"
-        self.fn_name = "Run Pipeline"
+        self.icon_type = "SchedulePipeline"
+        self.fn_name = "Schedule Pipeline"
```

Comment:
Wouldn't be better name: "TriggerPipeline"? (+ it will not run it immediately? if it would, the name RunPipeline would be nice